src/AppInstallerCLICore/Workflows/ManifestComparator.cpp

```diff
@@ -10,6 +11,28 @@ namespace AppInstaller::CLI::Workflow
 {
     bool InstallerComparator::operator() (const ManifestInstaller& installer1, const ManifestInstaller& installer2)
     {
+        // Applicable architecture should always come before inapplicable architecture
+        if (Utility::IsApplicableArchitecture(installer1.Arch) != -1 && Utility::IsApplicableArchitecture(installer2.Arch) == -1)

```

Comment:
>-1 [](start = 66, length = 2)

Can we have a constant defined for this?

Reply:
Can the constant not have s_?  A public name should be cleaner.

---
In reply to: [499894685](https://github.com/microsoft/winget-cli/pull/588#discussion_r499894685) [](ancestors = 499894685)